src/Decoder.c

```diff
@@ -1000,8 +1003,7 @@ static void ZydisSetOperandSizeAndElementInfo(const ZydisDecoderContext* context
                 operand->element_type = ZYDIS_ELEMENT_TYPE_INT;
             } else
             {
-                ZYAN_ASSERT(definition->size[context->eosz_index] ||
-                    (instruction->meta.category == ZYDIS_CATEGORY_AMX_TILE));
+                // ZYAN_ASSERT(definition->size[context->eosz_index] || (instruction->meta.category == ZYDIS_CATEGORY_AMX_TILE));
```

Comment:
Is this intentionally commented out? I feel like commented out asserts will confuse us in the future. I'd either reinstate it as a return condition or get rid of it entirely.

Reply:
It's still a valid assertion, but as `instruction` is now passed by the user, it might contain invalid data, so we can't rely on this one anymore. On the other hand I don't want to remove it entirely, because it caused an issue in the past. Might convert it to a plain comment.